docs/guides/ce-resource-management.rst

```diff
@@ -21,12 +21,41 @@ Required hardware resources are configured on a per-task basis by setting the ``
         ...
 
 ``resources`` expects a ``sdk.Resources()`` object that specifies some or all of:
-* number of cores (``cpu``)
-* amount of RAM (``memory``)
-* disk space (``disk``)
-* number of GPU computing units (``gpu``)
-
-The example above specifies a task that requires 100 milicores (or 0.1 cores), 1 Gb of RAM, 10 Gb of disk space, and access to a GPU.
+* ``cpu``: number of cores.
+* ``memory``: amount of RAM (bytes).
+* ``disk``: disk space (bytes).
+* ``gpu``: whether access to a gpu unit is required (``1`` if a GPU is required, ``0`` otherwise).
+
+Amounts of cpu and memory resources can be specified as a plain integer, or as a fixed-point float appended with the string representation of one of the following unit prefixes. These can be SI (metric), or IEC (binary) prefixes.
+
+.. table:: unit prefixes
+    :widths: auto
+
+    +---------+-------+--------+------------+----------------+-----------------+
+    |         | Name  | String | Power of 2 | Power of 1024  | Power of 10     |
+    +=========+=======+========+============+================+=================+
+    | Binary  | kibi  | ki     | 2^10       | 1024^1         | 1.024 x 10^3    |
```

Comment:
This is a capital "K"?
See https://github.com/kubernetes/apimachinery/blob/v0.26.2/pkg/api/resource/quantity.go#L54

Reply:
"I didn't choose the capitalization" lol
```suggestion
    | Binary  | kibi  | Ki     | 2^10       | 1024^1         | 1.024 x 10^3    |
```